src/software/ai/hl/stp/tactic/primitive_test.cpp

```diff
@@ -10,14 +10,15 @@ class PrimitiveTest : public testing::Test
    protected:
     RobotConstants_t robot_constants = create2021RobotConstants();
     Robot robot                      = TestUtil::createRobotAtPos(Point(0, 0));
-    World world                      = TestUtil::setFriendlyRobotPositions(
-        TestUtil::createBlankTestingWorld(), {Point(0, 0)}, Timestamp::fromSeconds(0));
+    std::shared_ptr<World> world     = TestUtil::createBlankTestingWorld();
     RobotNavigationObstacleFactory obstacle_factory =
         RobotNavigationObstacleFactory(TbotsProto::RobotNavigationObstacleConfig());
 };
 
 TEST_F(PrimitiveTest, test_create_move_primitive)
 {
+    TestUtil::setFriendlyRobotPositions(world, {Point(0, 0)}, Timestamp::fromSeconds(0));
+
```

Comment:
Curious why the `setFriendlyRobotPositions` calls were moved to the tests instead of being in the class definition?

Reply:
Just me being silly, fixed.